api/services/HelperService.js

```diff
@@ -850,6 +850,25 @@ var HelperService ={
         }).format(number);
     },
 
+    checkForIllegalCharacters(string) {
+        if (!string) {
+            sails.log.error('checkForIllegalCharacters: string argument is required');
```

Comment:
This is a good point. Since this is a helper function, I don't want to include page redirection logic so if there's an error it just returns a null. I don't have any logic on the function that uses this that handles null errors so I'll need to look into that 🤔